app/mailers/admin_mailer.rb

```diff
@@ -8,9 +8,9 @@ def error(exception, session = nil, params = nil, env = nil, user = nil)
     @params = params
     @env = env
     @user = user
-    @hostname = env['SERVER_NAME'] || env['HTTP_HOST'] || env['HTTP_ORIGIN']
+    @hostname = env['SERVER_NAME'] || env['HTTP_HOST'] || env['HTTP_ORIGIN'] if env
     @rails_env = Rails.env
-    path = (env && env['REQUEST_URI']) ? (": " + env['REQUEST_URI']) : ""
+    path = (env && env['REQUEST_URI']) ? (": " + env['REQUEST_URI']) : "" if env
```

Comment:
Existence of `env` is already checked in the initial condition. Not setting path would result in an error below.